.github/workflows/packaging.yml

```diff
@@ -77,10 +77,12 @@ jobs:
             -v "${PACKAGE_AUTOPSY_BIN_VERSION}" \
             --iteration "$PACKAGE_SLEUTHKIT_JAVA_REL" \
             --no-auto-depends \
-            -d "openjdk-8-jre" \
+            -d "bellsoft-java8-full" \
             -d "sleuthkit-java (>= ${PACKAGE_SLEUTHKIT_JAVA_VERSION})" \
             -d "testdisk" \
-            -d "openjfx" \
+            -d "ewf-tools" \
+            -d "xmount" \
+            -d "fdupes" \
```

Comment:
Can you explain why this is specifically needed? I don't need this on Arch. Might be good to know it and add on Arch too.

Reply:
This I copied from @mfrade autopsy install script. @mfrade can you comment on this ?

Reply:
Ok, I talked with @mfrade . Actually these packages are not needed, but imagemagick is.